discoverable-client/src/main/java/com/ca/mfaas/client/DiscoverableClientSampleApplication.java

```diff
@@ -18,19 +18,18 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.event.ApplicationReadyEvent;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.cloud.netflix.hystrix.HystrixAutoConfiguration;
 import org.springframework.context.ApplicationListener;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.web.socket.config.annotation.EnableWebSocket;
 
-@SpringBootApplication(exclude = HystrixAutoConfiguration.class)
+@SpringBootApplication/*(exclude = HystrixAutoConfiguration.class)*/
 @EnableApiDiscovery
 @EnableConfigurationProperties
 @EnableWebSocket
 @EnableApimlLogger
 @ComponentScan(value = {
     "com.ca.mfaas.client",
-    "com.ca.mfaas.enable",
+    "com.ca.apiml.enable",
```

Comment:
There is no need to define ```com.ca.apiml.enable```, in case you use ```@EnableApiDiscovery```

Reply:
Removed the ComponentScan.

Reply:
Actually it is needed. If not provided, RegisterToAPiLayer is not created and invoked.

Reply:
Let's check it again. That's purpose of ```@EnableApiDiscovery``` to enable spring components without definition of ```@ComponentScan```

Reply:
Fixed. Was missing @ComponentScan on the enabler Configuration class.